soak/README.md

```diff
@@ -1,317 +1,117 @@
-        kill -9 $pid ;
-    done
-    ```
\ No newline at end of file
+Create a new AWS account to house the following resources. A new account ensures
+that any leaked resources do not affect the e2e tests running on your pull
+requests, and that you can give the ACK core team permissions to view this
+account.
+
+Export the credentials for an AWS user with administrator access to your
+terminal before following the rest of the documents.
+
+### Step 1 (Run the bootstrapping script)
+
+Run the `bootstrap.sh` script in the current directory. This script will do the
+following: 
+1. Create an ECR public repository for the soak test runner
+1. Create an EKS cluster with the default soak test configuration (from
+  `cluster-config.yaml`)
+1. Install the controller Helm chart
+1. Install Prometheus and Grafana
+1. Install the custom ACK soak test Grafana dashboard (from
+   `./monitoring/grafana/ack-dashboard-source.json`)
+1. Build and push the soak test runner image
+1. Install the soak test runner Helm chart
+
+The script requires the name of the service and the semver tag of the controller
+version up for testing. For example:
+```bash
+./bootstrap.sh s3 v0.0.12
+```
+
+### Step 2 (Log onto the Prometheus dashboard)
+
+After the script concludes, it should provide an example of the command to
+port-forward Prometheus. By default, the command should look like the following:
+
+```bash
+kubectl port-forward -n prometheus service/kube-prom-grafana 3000:80 --address='0.0.0.0' >/dev/null &
+```
+
+Navigate to `http://127.0.0.1:3000/` and log in. You should be able to log into
+the dashboard with the following credentials:
+- Username: `admin`
+- Password: `prom-operator`
+
+Using the menu bar on the left, side navigate to `Dashboards` > `Browse`. Select
+the `ACK Dashboard` at the top of the list. The `ACK Dashboard` will show the
+ACK-specific request counts and error codes used by the controller when making
+calls to AWS APIs. 
+
+The `Kubernetes/Compute Resources/Pod` dashboard will show the resource
+consumption by the controller pod.
+
+### Step 3 (Notify the ACK core team)
+
+The ACK core team needs to be made aware that service controller is ready for
+soak test execution using Prow, which is a manual process. Therefore, once you
+have completed the above steps copy the IRSA ARN from the following command, and
+send it to a member of the ACK core contributor team:
+```bash
+kubectl get sa -n ack-system ack-core-account -o json | jq -r ".metadata.annotations.\"eks.amazonaws.com/role-arn\""
+```
+
+> **Note for Core Contributors:** Upon receiving a new IRSA ARN, access the
+ACK infrastructure account and add a new SSM string parameter with the path
+`/ack/prow/soak/irsa/<service>` and a value of the ARN.
+```bash
+# For ACK core contributors
+aws ssm put-parameter --name "/ack/prow/soak/irsa/$SERVICE" --type String
+--value <provided-value> 
+```
+
+### Step 4 (Customise your soak tests)
```

Comment:
I do not think `bootstrap.sh` consumes the `soak_config.yaml` file so this step is not valid.